### ISSUE TITLE
TTYVTDisallocate=no doesn't start tty1 automatically

### ISSUE THREAD

**Issue by vvtx** [systemd/systemd#6424]
### Submission type

 - Bug report

### systemd version the issue has been seen with

> 233

### Used distribution

> Arch Linux

### In case of bug report: Expected behaviour you didn't see

> I use the tty1 VT in other way that the rest ones. I don't want to clear the kernel and other messages from console on this tty so I set: TTYVTDisallocate=no in /etc/systemd/system/getty.target.wants/getty@tty1.service. 

### In case of bug report: Unexpected behaviour you saw

> The tty1 virtual terminal is not spawning login, so I cannot log on this VT. Only manual starting works: systemctl start getty@tty1.service.

### In case of bug report: Steps to reproduce the problem

> Instead of making soft links in /etc/systemd/system/getty.target.wants/ directory just copy the target file from /usr/lib/systemd/system/getty@.service and name it getty@tty1.service. Set following: TTYVTDisallocate=no in it and reboot the system.

**Comment by poettering** [comment]
I am not sure I grok what you are trying to do. But do notw, that .wants/ directories should only contain symlinks, not files. hence, if you set TTYVTDisallocate= then that's not going to be respected in all cases.

Also, note that tty1 is managed differently from other ttys by logind: you have to explicitly run a getty or gdm on it, logind won't handle that automatically for you.

Normally it should suffice to do:

```
mkdir -p /etc/systemd/system/getty@tty1.service.d
cat > /etc/systemd/system/getty@tty1.service.d/50-vtdisallocate.conf <<EOF
[Service]
TTYVTDisallocate=no
EOF
```

This will simply extend the existing unit file with this one setting, instead of overriding it.

**Comment by vvtx** [comment]
Ok, I didn't know that getty.target.wants should only have symlinks, so I placed a file with the contents from linked file (targeted), changed TTYVTDisallocate to "no" and it has been working so far.  
The reason that I used TTYVTDisallocate=no was that I wanted to keep all the messages on console tty1 from Linux boot process. Opposite setting "TTYVTDisallocate=yes" clears the screen before spawning "login" - and I didn't want that behavior.

I have tested now your solution - and seems like it is working at the moment, however it takes a few seconds to spawn "login".
